### ISSUE TITLE
\ProvidesClass breaks the build

### ISSUE THREAD

**Issue by sheepx86** [NVSL/libpaper#17]
line 3: `\ProvidesClass{arxiv}[NVSL ArXiv submission class]` in `arxiv.cls` breaks the build of the 3DXPoint paper.

**Comment by sheepx86** [comment]
It seems the same issue still exist on the build server (Debian 9).
It build without any issue when the line is commented.
```
(cd './libpaper/bin/'; ./check_repo)
'./libpaper/bin/'/check_release > libpaper_metadata.tex || true
TEXINPUTS=./:./libpaper//: pdflatex -interaction=errorstopmode -halt-on-error paper 
This is pdfTeX, Version 3.14159265-2.6-1.40.17 (TeX Live 2016/Debian) (preloaded format=pdflatex)
 restricted \write18 enabled.
entering extended mode
(./paper.tex
LaTeX2e <2017/01/01> patch level 3
Babel <3.9r> and hyphenation patterns for 83 language(s) loaded.
(./libpaper/arxiv.cls (/usr/share/texlive/texmf-dist/tex/latex/base/article.cls
Document Class: article 2014/09/29 v1.4h Standard LaTeX document class
(/usr/share/texlive/texmf-dist/tex/latex/base/size10.clo))
Document Class: arxiv NVSL ArXiv submission class
(/usr/share/texlive/texmf-dist/tex/latex/psnfss/times.sty)
(/usr/share/texlive/texmf-dist/tex/latex/geometry/geometry.sty
(/usr/share/texlive/texmf-dist/tex/latex/graphics/keyval.sty)
(/usr/share/texlive/texmf-dist/tex/generic/oberdiek/ifpdf.sty)
(/usr/share/texlive/texmf-dist/tex/generic/oberdiek/ifvtex.sty)
(/usr/share/texlive/texmf-dist/tex/generic/ifxetex/ifxetex.sty))
(/usr/share/texlive/texmf-dist/tex/latex/fancyhdr/fancyhdr.sty)

Package Fancyhdr Warning: \fancyfoot's `E' option without twoside option is use
less on input line 39.


Package Fancyhdr Warning: \fancyfoot's `E' option without twoside option is use
less on input line 40.

)
! Missing number, treated as zero.
<to be read again> 
                   N
l.3 \pdfoutput
              =1  % useful for submission to ArXiv
!  ==> Fatal error occurred, no output PDF file produced!
Transcript written on paper.log.
libpaper/Make.rules:67: recipe for target 'paper.aux' failed
make: *** [paper.aux] Error 1

```

**Comment by sheepx86** [comment]
Fixed in commit cf45854391eacd4bed3b79d50ec5797aeac0e23f
